indico/modules/events/notifications.py

```diff
@@ -13,6 +15,19 @@
 from indico.web.flask.templating import get_template_module
 
 
+def _get_emails_from_category(category):
+    emails = set(category.event_creation_notification_emails)
+
+    if category.notify_managers:
```

Comment:
I think we should ignore this setting when it comes to move request notifications...